shippable.yml

```diff
@@ -18,31 +18,6 @@ matrix:
     - env: T=units/3.8/1
     - env: T=units/3.9/1
 
-    - env: T=windows/2012/1
-    - env: T=windows/2012-R2/1
-    - env: T=windows/2016/1
-    - env: T=windows/2019/1
-
-    - env: T=windows/2012/2
-    - env: T=windows/2012-R2/2
-    - env: T=windows/2016/2
-    - env: T=windows/2019/2
-
-    - env: T=windows/2012/3
-    - env: T=windows/2012-R2/3
-    - env: T=windows/2016/3
-    - env: T=windows/2019/3
-
-    - env: T=windows/2012/4
-    - env: T=windows/2012-R2/4
-    - env: T=windows/2016/4
-    - env: T=windows/2019/4
-
-    - env: T=windows/2012/5
-    - env: T=windows/2012-R2/5
-    - env: T=windows/2016/5
-    - env: T=windows/2019/5
-
```

Comment:
@joezollo My apologies. When I saw you had configured Shippable on your fork and were pushing commits to your `master` branch, the thought hadn't occurred to me that you might have an open PR from that branch. Since you're using your repo for this PR instead of just testing in your own repo, the correct thing to do is:

1) Disable Shippable on your repo. You should be able to do that here: https://app.shippable.com/subs/github/joezollo/enable
2) Revert this modification to the `shippable.yml` file.

That will allow you to continue with your PR while avoiding the issue I originally reported here: https://github.com/joezollo/community.windows/pull/1.

In retrospect I should have checked for a PR here first.

Reply:
No worries @mattclay - let me work on this.